.gitignore

```diff
@@ -4,6 +4,7 @@ prometheus-sql.exe
 .idea
 .vagrant
 vendor
+vendor.orig
```

Comment:
Any reason for adding this? Seems like a temporary inclusion based on this transition for your local repo.